src/big/lib_gmp.cr

```diff
@@ -84,6 +84,13 @@ lib LibGMP
   fun cmp_ui = __gmpz_cmp_ui(op1 : MPZ*, op2 : ULong) : Int
   fun cmp_d = __gmpz_cmp_d(op1 : MPZ*, op2 : Double) : Int
 
+  # # Number Theoretic Functions
+
+  fun gcd = __gmpz_gcd(rop : MPZ*, op1 : MPZ*, op2 : MPZ*)
+  fun gcd_ui = __gmpz_gcd_ui(rop : MPZ*, op1 : MPZ*, op2 : MPZ*) : Int
```

Comment:
This binding (and lcm_ui) should have op2 be `ULong`: https://gmplib.org/manual/Number-Theoretic-Functions.html#index-mpz_005fgcd_005fui